components/inputs/input-date.js

```diff
@@ -216,7 +216,7 @@ class InputDate extends LocalizeStaticMixin(LitElement) {
 					@mouseup="${this._handleMouseup}"
 					placeholder="${shortDateFormat}"
 					style="${styleMap({maxWidth: inputTextWidth})}"
-					title="${this.localize('openInstructions', {format: shortDateFormat})}"
+					title="${this.localize('inputDate.openInstructions', {format: shortDateFormat})}"
```

Comment:
Wondering if it should be `components.inputs.date.openInstructions` in terms of namespacing? We can have as many dots as we want right? Just thinking about when `templates` have translations, or helpers, mixins, etc.